pilot/pkg/model/config_test.go

```diff
@@ -243,36 +244,41 @@ func TestResolveShortnameToFQDN(t *testing.T) {
 func TestMostSpecificHostMatch(t *testing.T) {
 	tests := []struct {
 		in     []host.Name
+		m      map[host.Name]struct{}
```

Comment:
nit: all the tests pass empty list here. should just construct it in the test?

Reply:
👌，resolved #28679